### PR TITLE
#1031 check if PR description contains 'fixes #issue'

### PR DESCRIPTION
**Pull request by jstrzebonski**
Related to DARMA-tasking/vt#1031

Things in `dist/` directory are generated automatically.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: Check PR comment
```
Comment (jstrzebonski) [inline action.yml]: Check PR description

File: action.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+author: jstrzebonski
+branding:
+  icon: "search"
+  color: "green"
+description: PR comment
```
Comment (jstrzebonski) [inline action.yml]: PR description

File: action.yml (hunk 1 above)
Comment (jstrzebonski) [reply action.yml]: Done.

File: action.yml (hunk 2 above)
Comment (jstrzebonski) [reply action.yml]: Done.

File: src/helpers.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+function checkPrTitle(prTitle) {
+  let prTitleRegexp = /^#?\d+\s+.+$/;
```
Comment (jstrzebonski) [inline src/helpers.js:7]: Should I comment on this and the others regexps to explain them?

File: README.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+## Development
+
+Install the dependencies
+
+```bash
+npm install
+```
+
+Run the tests
```
Comment (jstrzebonski) [inline README.md:35]: Add note about linting

File: README.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+git add dist/
+```
+
+GitHub Actions will run the entry point from the action.yml
+
+```yml
+runs:
+  using: "node12"
+  main: "dist/index.js" # <== entry point
+```
```
Comment (jstrzebonski) [inline README.md:84]: Add note about `npm run all`

**Comment by cz4rs** [comment]
> Related to [DARMA-tasking/vt#1031](https://github.com/DARMA-tasking/vt/issues/1031)

You missed the opportunity to use "fixes" yourself ;)

File: README.md (hunk 4 above)
Comment (jstrzebonski) [reply README.md:35]: Done.

File: README.md (hunk 5 above)
Comment (jstrzebonski) [reply README.md:84]: Done.

File: src/helpers.js (hunk 3 above)
Comment (lifflander) [reply src/helpers.js:7]: No, I think these are self explanatory

**Review by lifflander — APPROVED** [review]

**Review by lifflander — COMMENTED** [review]
Can you add the other keywords that automatically close issues also?

Please refer to: https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

**Review by lifflander — CHANGES_REQUESTED** [review]
Everything looks great here! If you can just add the other keywords for closing issues that would be great!

**Comment by lifflander** [comment]
@jstrzebonski I'm also thinking it might be nice to check the branch name to ensure it contains the right issue number, with the proper format `123-description-of-branch`. I noticed that some people have added the `#` before the branch name which is not the proper format. So it would be nice if we can check that pattern also!

**Comment by jstrzebonski** [comment]
@lifflander 
✔️ All GitHub's issue closing keywords are supported (close, fix, resolve).
✔️ Checking format of branch name (starting from issue number, without any hash decoration) is done.
✔️  Checking if branch, title and description points to the same issue is done.

❓  Quick question - I see that there are `feature-...` branches in DARMA/vt. Should we support it? I mean right know I'm checking if branch start with a number, so feature branches will make the test fail.

**Comment by lifflander** [comment]
> @lifflander
> ✔️ All GitHub's issue closing keywords are supported (close, fix, resolve).
> ✔️ Checking format of branch name (starting from issue number, without any hash decoration) is done.
> ✔️ Checking if branch, title and description points to the same issue is done.
> 
> ❓ Quick question - I see that there are `feature-...` branches in DARMA/vt. Should we support it? I mean right know I'm checking if branch start with a number, so feature branches will make the test fail.

No, let's not support the branch names that start with `feature-...`. That was from a long time ago.

**Review by lifflander — APPROVED** [review]

**Comment by lifflander** [comment]
This is good to merge!

**Pull request merged**